packages/mongo/collection.js

```diff
@@ -1240,13 +1240,13 @@ function popCallbackFromArgs(args) {
   }
 }
 
-ASYNC_COLLECTION_METHODS.forEach(methodName => {
-  const methodNameAsync = getAsyncMethodName(methodName);
-  Mongo.Collection.prototype[methodNameAsync] = function(...args) {
-    try {
-      return Promise.resolve(this[methodName](...args));
-    } catch (error) {
-      return Promise.reject(error);
-    }
-  };
-});
+// ASYNC_COLLECTION_METHODS.forEach(methodName => {
```

Comment:
There should be at least a comment here if this is not being removed completely, so that if someone stumbles upon this they are aware of why this was done.

Reply:
I agree